docs/src/pages/lab/slider/CustomIconSlider.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Typography from '@material-ui/core/Typography';
+import Slider from '@material-ui/lab/Slider';
+import StarsIcon from '@material-ui/icons/Gamepad';
```

Comment:
`GamepadIcon`?

Reply:
Oops! Updating it :P

Reply:
I have updated it to Lens icon, which is basically a filled circle.